spec.bs

```diff
@@ -842,6 +842,7 @@ The following data structures are shared between registration and authentication
     dictionary PaymentCredentialInstrument {
         required DOMString displayName;
         required USVString icon;
+                 USVString alt;
```

Comment:
This should probably be a `DOMString` unless you have a good reason for using a `USVString`.